libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/GuidanceViewUrlMatcher.kt

```diff
@@ -0,0 +1,32 @@
+@file:JvmName("GuidanceViewUrlMatcher")
+
+package com.mapbox.navigation.ui.maps.guidance
+
+import com.mapbox.api.directions.v5.models.BannerComponents
+
+internal fun String.guidanceViewUrlMatcher(
+    type: String,
+    accessToken: String,
+    key: String = "access_token=",
+): String {
+    when (type) {
+        BannerComponents.JCT -> {
+            return if (this.endsWith('&')) {
+                this.plus(key.plus(accessToken))
+            } else if (this.endsWith('?')) {
+                this.replace('?', '&').plus(key.plus(accessToken))
```

Comment:
`?` already marks the first query parameter, replacing it would break the URL.

Reply:
For junction views, Nav ApI network request demands `&access_token=value` whereas for signboards and other subTypes it demands `?access_token=value`

I have raised this earlier with the directions team to keep it consistent.

Reply:
Could you provide example URLs? This is very strange.